tests/helpers/test_service.py

```diff
@@ -564,6 +564,20 @@ async def test_async_get_all_descriptions(hass: HomeAssistant) -> None:
     assert "description" in descriptions[logger.DOMAIN]["set_level"]
     assert "fields" in descriptions[logger.DOMAIN]["set_level"]
 
+    hass.services.async_register(logger.DOMAIN, "new_service", lambda x: None, None)
+    service.async_set_service_schema(
+        hass, logger.DOMAIN, "new_service", {"description": "new service"}
+    )
+    descriptions = await service.async_get_all_descriptions(hass)
+    assert "description" in descriptions[logger.DOMAIN]["new_service"]
+    assert descriptions[logger.DOMAIN]["new_service"]["description"] == "new service"
+
```

Comment:
Since it’s cached, should we test that with

```python
assert await service.async_get_all_descriptions(hass) is descriptions
```

Reply:
Sounds good

3b3edbac1ffa31e462b2fd9c2525a2ca7cde8de3